applications_tests/gd_navier_stokes_2d/apparent_viscosity_carreau_gd.mpirun=2.output

```diff
@@ -0,0 +1,13 @@
+Running on 2 MPI rank(s)...
+   Number of active cells:       16
+   Number of degrees of freedom: 187
+   Volume of triangulation:      1
+Apparent viscosity: 0.781234 Pa.s
+
+*****************************************************************
+Steady iteration :        1/1
+*****************************************************************
+Apparent viscosity: 0.781234 Pa.s
+L2 error velocity : 5.19289e-07
+cells error_velocity error_pressure 
+   16 5.192891e-07 - 4.658475e-03 - 
```

Comment:
Can we check that the apparent viscosity converges with the mesh?

Reply:
I don't think there is a tool to check post-processing variable convergence with an analytical solution (@blaisb ?) but I just checked it on the side and it converges to the same rate as the velocity when refining twice.

Reply:
That would be difficult to add to the convergence table (although doable). Not sure if it is important though :).